packages/cli/package.json

```diff
@@ -71,6 +71,7 @@
     "eslint-plugin-jest": "^24.1.0",
     "eslint-plugin-jsx-a11y": "^6.2.1",
     "eslint-plugin-monorepo": "^0.3.2",
+    "eslint-plugin-notice": "^0.9.10",
```

Comment:
Eh, realizing that we shouldn't be shipping this with the CLI, it should be installed in the root of the repo instead. This check is specific to the main repo only

Reply:
Yeah that's totally right, good we didn't merge this before fixing that :)